benchmarking/definitions/definition_nashpobench.py

```diff
@@ -10,31 +10,30 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
+from syne_tune.search_space import choice, logfinrange, finrange
 
-from sklearn.ensemble import RandomForestRegressor
-
-from syne_tune.search_space import choice, randint, uniform, loguniform
-
-from benchmarking.blackbox_repository.conversion_scripts.scripts.fcnet_import import \
-    METRIC_ELAPSED_TIME, METRIC_VALID_LOSS, RESOURCE_ATTR, BLACKBOX_NAME
+from benchmarking.blackbox_repository.conversion_scripts.scripts.fcnet_import \
+    import METRIC_ELAPSED_TIME, METRIC_VALID_LOSS, RESOURCE_ATTR, BLACKBOX_NAME
 
 
+# This configuration space allows to use the tabulated blackbox without any
+# interpolation (surrogate), meaning that all numerical HPs have finite ranges.
+# Note that `hp_init_lr` remains categorical and will be 1-hot encoded. It
+# would be better for BO to use a surrogate.
 _config_space = {
     "hp_activation_fn_1": choice(["tanh", "relu"]),
     "hp_activation_fn_2": choice(["tanh", "relu"]),
-    "hp_batch_size": randint(8, 64),
-    "hp_dropout_1": uniform(0.0, 0.6),
-    "hp_dropout_2": uniform(0.0, 0.6),
-    "hp_init_lr": loguniform(0.0005, 0.1),
+    "hp_batch_size": logfinrange(8, 64, 4, is_int=True),
```

Comment:
If we update this one, we should also update `fcnet_recipe.py` for consistency with the same values.

Reply:
Good idea